crates/core_arch/src/riscv/mod.rs

```diff
@@ -0,0 +1,10 @@
+//! RISC-V intrinsics
+
+/// Generates the `PAUSE` instruction
+///
+/// The PAUSE instruction is a HINT that indicates the current hart's rate of instruction retirement
+/// should be temporarily reduced or paused. The duration of its effect must be bounded and may be zero.
+#[inline(always)]
```

Comment:
Is there a specific reason for using `#[inline(always)]`. A normal `#[inline]` should be sufficient.

Reply:
Hello! I suspect if a single instruction wrap would gain better performance if it inlines. Yes, for some debug purposes we should not inline these functions. and `#[inline]` only would behave better in these scenearios.